### PR TITLE
Address build issues occurring when building with Library Evolution support, Xcode 16

### PR DESCRIPTION
**Pull request by gemmakbarlow**
Address build issues seen building for library evolution mode, Xcode 16 - i.e. via `make build-for-library-evolution`.

image

### CODE REVIEW

File: Sources/IssueReporting/ReportIssue.swift [hunk 1]
```diff
@@ -73,6 +58,25 @@ public func reportIssue(
...
+      line: Int(IssueContext.current?.line ?? line),
+      column: Int(IssueContext.current?.column ?? column)
+    )
+  case .xcTest:
+    _XCTFail(
+      message().withAppHostWarningIfNeeded() ?? "",
+      file: IssueContext.current?.filePath ?? filePath,
+      line: IssueContext.current?.line ?? line
+    )
+  @unknown default: break
```
Comment (stephencelis) [inline Sources/IssueReporting/ReportIssue.swift:79]: Do you think it's worth reporting issues from this branch? From a practical perspective I can't see this unknown branch ever hitting, though, so this should be OK.

**Review by stephencelis — APPROVED** [review]
Thanks! We actually had a library evolution CI action that was still on 15.4, so I've pushed an update for it to run against Xcode 16. If things go green I'll merge!

**Comment by gemmakbarlow** [comment]
> Thanks! We actually had a library evolution CI action that was still on 15.4, so I've pushed an update for it to run against Xcode 16. If things go green I'll merge!

Oh that makes sense 😄 Awesome.

**Comment by gemmakbarlow** [comment]
@stephencelis - if you're able to cut a new release and update some of the other `pointfreeco` SDKs to include the fix, I'd appreciate it (some build issues stemming from this currently causing me pain) 🙏🏻 

Happy to help with the effort if useful !

**Comment by stephencelis** [comment]
@gemmakbarlow Done! https://github.com/pointfreeco/swift-issue-reporting/releases/tag/1.4.1

There should be no need to change our other libraries to receive the update. Just make sure you update your packages in you project so that the Package.resolved file pulls in the newer version and you should be good to go! Or you can explicitly depend on 1.4.1 in your package/project.

**Comment by gemmakbarlow** [comment]
> @gemmakbarlow Done! https://github.com/pointfreeco/swift-issue-reporting/releases/tag/1.4.1
> 
> There should be no need to change our other libraries to receive the update. Just make sure you update your packages in your project so that the Package.resolved file pulls in the newer version and you should be good to go! Or you can explicitly depend on 1.4.1 in your package/project.

Perfect, thanks for the tip and speedy replies.

**Pull request merged**